src/App.vue

```diff
@@ -1,11 +1,53 @@
 <template>
-
+  <div id="presentation-viewer-main" class="presentation-viewer oc-flex">
+    <div class="reveal">
+      <div class="slides">
+        <section :data-markdown="props.url" :data-separator="dataSeparator"
+                 :data-separator-vertical="dataSeparatorVertical"></section>
```

Comment:
`dataSeparator` and `dataSeparatorVertical` are defined as props and I wonder if this works.
```suggestion
        <section :data-markdown="props.url" :data-separator="dataSeparator"
                 :data-separator-vertical="dataSeparatorVertical"></section>
```

Reply:
Those custom props work fine. But I don't know if it is the right way or not.